### DISCUSSION TITLE
Improve Timepicker user experience

### DISCUSSION THREAD

**Discussion by jbcbezerra** [primefaces/community#3128] [PrimeNG]
The current timepicker in p-calendar is not usable. If i want to set a specific minute, i would have to click the arrow buttons until i get it, instead of just having an input field where i can set my desired minutes.
This issue was already opened multiple time but never have there be any work to it, so i am opening it here again.
https://github.com/primefaces/primeng/issues/1883

Is there any plan to improve it ?

**Comment by jbcbezerra** [comment]
After a year of no answer i am checking that this is still an unsolved problem in v19
Could you tell me the status of that @mertsincan ?

**Reply by Viktor-Ivliev** [reply]
https://github.com/orgs/primefaces/discussions/4577
